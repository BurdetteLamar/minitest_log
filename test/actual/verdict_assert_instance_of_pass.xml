<log>
  <summary_ verdicts='2' failures='0' errors='0'/>
  <section_ name='Test' method='verdict_assert_instance_of?' type='pass'>
    <section_ name='Args=[Integer, 0, &quot;Message 0&quot;]'>
      <verdict_ method='verdict_assert_instance_of?' outcome='passed' id='0' message='Message 0'>
        <expected_ class='Class' value='Integer'/>
        <actual_ class='Integer' value='0'/>
      </verdict_>
    </section_>
    <section_ name='Args=[Symbol, :a, &quot;Message 1&quot;]'>
      <verdict_ method='verdict_assert_instance_of?' outcome='passed' id='1' message='Message 1'>
        <expected_ class='Class' value='Symbol'/>
        <actual_ class='Symbol' value=':a'/>
      </verdict_>
    </section_>
  </section_>
</log>
