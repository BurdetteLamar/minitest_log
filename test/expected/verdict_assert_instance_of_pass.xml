<log>
  <summary_ verdicts='3' failures='0' errors='0'/>
  <section_ name='Test' method='verdict_assert_instance_of?' type='pass'>
    <section_ name='Args=[Integer, 0, &quot;Message 0&quot;]'>
      <verdict_ method='verdict_assert_instance_of?' outcome='passed' id='0' message='Message 0'>
        <expected_ class='Class' value='Integer'/>
        <actual_ class='Integer' value='0'/>
      </verdict_>
    </section_>
    <section_ name='Args=[Symbol, :a, &quot;Message 1&quot;]'>
      <verdict_ method='verdict_assert_instance_of?' outcome='passed' id='1' message='Message 1'>
        <expected_ class='Class' value='Symbol'/>
        <actual_ class='Symbol' value=':a'/>
      </verdict_>
    </section_>
  </section_>
  <verdict_ method='verdict_assert_equal?' outcome='passed' id='error_count'>
    <expected_ class='Integer' value='0'/>
    <actual_ class='Integer' value='0'/>
  </verdict_>
</log>
